machinist/polling/register.go

```diff
@@ -25,12 +26,18 @@ func SendRegisterRequests(ctx context.Context, client machinist_rpc.ControllerCl
 	}
 	for {
 		if err := pollStream.Send(registerRequest); err != nil {
-			l.Errorf("unable to send request: %w", err)
+			_, err := pollStream.Recv()
+			s, ok := status.FromError(err)
+			if ok {
+				l.Errorf("unable to send register request: %s %s", s.Code(), s.Message())
```

Comment:
Does %w and passing err directly not log both the status and message?

Reply:
It does, I just was just doing it so it woudl log prettier. have no issue %w ing.